src/query/storages/fuse/Cargo.toml

```diff
@@ -56,4 +56,6 @@ streaming-decompression = "0.1.2"
 tokio-rayon = "2.1.0"
 tracing = "0.1.36"
 typetag = "0.2.3"
-uuid = { version = "1.1.2", features = ["serde", "v4"] }
+zstd = "0.12.3"
+bincode = "1.3.3"
```

Comment:
bincode is almost ready for `2.0.0`. It's better for us to use `v2.0.0-rc.3` for now.

Reply:
Thanks a lot! I will modify it:D

Reply:
2.0.0 need add new additional annotations,
#[derive(Encode, Decode, PartialEq, Debug)]